src/emailservice/requirements.txt

```diff
@@ -12,12 +12,12 @@ google-api-python-client==1.10.0  # via google-cloud-profiler
 google-auth-httplib2==0.0.4  # via google-api-python-client, google-cloud-profiler
 google-auth==1.20.1       # via google-api-core, google-api-python-client, google-auth-httplib2, google-cloud-profiler
 google-cloud-core==1.3.0  # via google-cloud-trace
-google-cloud-monitoring==0.36.0  # via opencensus-ext-stackdriver
+google-cloud-monitoring==0.36.0  # via opencensus-ext-stackdriver, opentelemetry-exporter-google-cloud
 google-cloud-profiler==1.1.2  # via -r requirements.in
-google-cloud-trace==0.24.0  # via -r requirements.in, opencensus-ext-stackdriver
+google-cloud-trace==0.23.0  # via -r requirements.in, opencensus-ext-stackdriver, opentelemetry-exporter-google-cloud
 googleapis-common-protos==1.52.0  # via google-api-core
-grpcio-health-checking==1.31.0  # via -r requirements.in
-grpcio==1.31.0            # via -r requirements.in, google-api-core, grpcio-health-checking, opencensus-ext-grpc
+grpcio-health-checking==1.30.0  # via -r requirements.in
+grpcio==1.30.0            # via -r requirements.in, google-api-core, grpcio-health-checking, opencensus-ext-grpc, opentelemetry-ext-grpc
```

Comment:
note that the slightly older versions here are selected for the opentelemetry dependencies